adafruit_wiznet5k/__init__.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: 2023 Martin Stephens
+#
+# SPDX-License-Identifier: MIT
+
+"""Makes a debug message function available to all modules."""
+try:
+    from typing import TYPE_CHECKING, Optional, Union, Tuple, Sequence
+
+    if TYPE_CHECKING:
+        from adafruit_wiznet5k.adafruit_wiznet5k import WIZNET5K
+except ImportError:
+    pass
+
+import gc
+
+
+def debug_msg(
```

Comment:
I'd prefer if __init__.py was only used for initialization and configuration, not to declare methods.

Could you instead pull the helper functions into `_wiznet5k.py` or another file?

Reply:
I'll move it to something like `wiznet5k_debug.py` then. It'll be imported by each module that has debugging, eventually.

Reply:
OK, I've moved it from `init.py` to `adafruit_wiznet5k.adafruit_wiznet5k_debug.py`